packages/zero-client/src/client/mutation-tracker.ts

```diff
@@ -96,7 +96,10 @@ export class MutationTracker {
 
   trackMutation(): MutationTrackingData {
     const id = nextEphemeralID();
-    const mutationResolver = resolver<MutationOk, ErrorType>();
+    const mutationResolver = resolver<void, ErrorType>();
+
+    // prevent unhandled rejection warnings if callers ignore the server promise
+    mutationResolver.promise.catch(() => {});
```

Comment:
If we are going to catch / blackhole these, we should at least create an `onError` callback on the zero-client to allow catching these. @aboodman ?